<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButtonguass">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>高斯噪声</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonsalt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>80</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>椒盐噪声</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetRaw" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widgetOut" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
